### PR TITLE
Target release 6/20: Android agent v7.4.0 release notes and related docs

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? - android release note v7.4.0 and all related docs

### CODE REVIEW

**Comment by homelessbirds** [comment]
@ywang-nr let us know when you're ready for a review :)

**Comment by ywang-nr** [comment]
> @ywang-nr let us know when you're ready for a review :)

Thank you, this PR will be draft for some time as it's for our next release, which would happen in another month-ish.

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+subject: Android agent
+releaseDate: '2024-06-20'
+version: 7.4.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.4.0.zip'
+---
+
+## New
+
+* Add ApplicationExitInfo as part of the data reporting. Check out more details of how to enable this feature [here](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-applicationexitinfo-reporting).
```
Comment (akristen) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx:10]:
```suggestion
* Add `ApplicationExitInfo` as part of data reporting. You can read our docs to learn how to [enable this feature](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-applicationexitinfo-reporting).
```

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* Add the ability to continue reporting data while application goes into the background state. Check out more details of how to enable this feature [here](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-background-reporting).
```
Comment (akristen) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx:11]:
```suggestion
* Add the ability to report data while an application goes into a background state. You can read our docs to learn how to [enable this feature](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#android-background-reporting).
```

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* Includes code in preparation for upcoming Log Forwarding feature, see your Account Team for information on early adoption.
```
Comment (akristen) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx:12]:
```suggestion
* Includes code for upcoming log forwarding feature. See your Account Team for information about early adoption.
```

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Fix
+
+* Enhance setUserId() function to add the ability to update userID attribute multiple times in a short period without losing the history of different end users. Check out more details of this fix [here](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-custom-user-id/#android).
```
Comment (akristen) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx:16]:
```suggestion
* Enhance `setUserId()` function to add the ability to update userID attribute multiple times in a short period without losing the history of different end users. You can read our docs to [learn about this fix](docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-custom-user-id/#android).
```

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+* Enhance the accuracy of device time calculation when it is changed by the mobile network.
+* Resolve an ANR issue when the application reports events.
+* Resolve an issue when exceptionMessage of a MobileHandledException doesn't display in query results.
+
+## Support statement
+
+We recommend you to upgrade the agent at least every 3 months. The specific policies and dates for Android agent support of the can be found in the [Mobile monitoring agents EOL policy](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy/#android-eol).
```
Comment (akristen) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-740.mdx:23]:
```suggestion
We recommend upgrrading the agent at least every 3 months. The specific policies and dates for Android agent support of the can be found in the [Mobile monitoring agents EOL policy](/docs/mobile-monitoring/new-relic-mobile/get-started/mobile-agents-eol-policy/#android-eol).
```

**Review by akristen — APPROVED** [review]

**Pull request merged**